.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: 🐞 Report a bug
+description: Create a bug report about unexpected behavior.
+labels: [kind/bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please only use this form to submit problems with building Kraftkit or it's interaction with Unikraft unikernels.  This includes unexpected or undefined behavior, stacktraces, critical messages, and things just breaking or exploding.
+        
+        **Ensure you have tested against the latest changes on `staging` branch before submitting this form.**
+
+        For questions, concerns or help with running Kraftkit, please check out our additional resources:
+
+        * [Unikraft Documentation](https://unikraft.org/docs/)
+        * [Unikraft Discussions Forum](https://github.com/unikraft/kraftkit/discussions)
```

Comment:
We don't have a discussions forum for kraftkit, so we can switch it back to the unikraft core repo for discussions